openfisca_aotearoa/variables/acts/citizenship/citizenship.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+from openfisca_core.model_api import *
+from openfisca_aotearoa.entities import Person
+
+
+class citizenship__present_for_min_1350_days_during_5_years(Variable):
+    value_type = bool
+    entity = Person
+    definition_period = MONTH
+    label = u"Applicant was present in New Zealand for a min of 1,350 days during the 5 years immediately preceding the date of application"
+    reference = "http://www.legislation.govt.nz/act/public/1977/0061/latest/DLM443855.html"
```

Comment:
We'll need to add the calculation here -- openfisca has the days they were here and the days they weren't, and needs to work out if that was enough